lua/apisix/http/router/radixtree_host_uri.lua

```diff
@@ -0,0 +1,180 @@
+-- Copyright (C) bobo pan
+
+local require = require
+local router = require("resty.radixtree")
+local core = require("apisix.core")
+local plugin = require("apisix.plugin")
+local ipairs = ipairs
+local type = type
+local str_reverse = string.reverse
+local error = error
+local user_routes
+local cached_version
+
+
+local _M = {version = 0.1}
+
+    local only_uri_routes = {}
```

Comment:
`only_uri_routes`, `only_uri_router `, `host_uri_routes ` and `host_uri_router` shoud be a top level variable.